irs-policy-store/src/main/java/org/eclipse/tractusx/irs/policystore/controllers/PolicyStoreController.java

```diff
@@ -217,6 +218,42 @@ public void deleteAllowedPolicy(@PathVariable("policyId") final String policyId)
         service.deletePolicy(policyId);
     }
 
+    @Operation(operationId = "removeAllowedPolicyFromBpnl",
+               summary = "Removes a policy from BPNL that should no longer be accepted in EDC negotiation.",
+               security = @SecurityRequirement(name = "api_key"), tags = { "Item Relationship Service" },
+               description = "Removes a policy from BPNL that should no longer be accepted in EDC negotiation.")
+    @ApiResponses(value = { @ApiResponse(responseCode = "200"),
+                            @ApiResponse(responseCode = "400", description = "Policy deletion failed.",
+                                         content = { @Content(mediaType = APPLICATION_JSON_VALUE,
+                                                              schema = @Schema(implementation = ErrorResponse.class),
+                                                              examples = @ExampleObject(name = "error",
+                                                                                        ref = "#/components/examples/error-response-400"))
+                                         }),
+                            @ApiResponse(responseCode = "401", description = UNAUTHORIZED_DESC,
+                                         content = { @Content(mediaType = APPLICATION_JSON_VALUE,
+                                                              schema = @Schema(implementation = ErrorResponse.class),
+                                                              examples = @ExampleObject(name = "error",
+                                                                                        ref = "#/components/examples/error-response-401"))
+                                         }),
+                            @ApiResponse(responseCode = "403", description = FORBIDDEN_DESC,
+                                         content = { @Content(mediaType = APPLICATION_JSON_VALUE,
+                                                              schema = @Schema(implementation = ErrorResponse.class),
+                                                              examples = @ExampleObject(name = "error",
+                                                                                        ref = "#/components/examples/error-response-403"))
+                                         }),
+    })
+    @DeleteMapping("/policies/{policyId}/bpnl/{bpnl}")
+    @ResponseStatus(HttpStatus.OK)
+    @PreAuthorize("hasAuthority('" + IrsRoles.ADMIN_IRS + "')")
+    public void removeAllowedPolicyFromBpnl(
+            // TODO (#528): add validation
+            //@ValidPolicyId
```

Comment:
Remove this TODO since there is already a PR open for #528 where this is addressed

Reply:
```suggestion
```